File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/domain/Models/ConferenceEditableInfo.cs
using System;

namespace ConferenceManagement
{
    public class ConferenceEditableInfo
    {
        public string Name { get; private set; }
        public string Description { get; private set; }
        public string Location { get; private set; }
        public string Tagline { get; private set; }
        public string TwitterSearch { get; private set; }
        public DateTime StartDate { get; private set; }
        public DateTime EndDate { get; private set; }

        public ConferenceEditableInfo() { }
        public ConferenceEditableInfo(string name, string description, string location, string tagline, string twitterSearch, DateTime startDate, DateTime endDate)
        {
            Name = name;
            Description = description;
            Location = location;
            Tagline = tagline;
            TwitterSearch = twitterSearch;
            StartDate = startDate;
            EndDate = endDate;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/ConferenceEditableInfo.java
package com.microsoft.conference.management.domain.Models;

import java.util.Date;

public class ConferenceEditableInfo {
    public String Name;
    public String Description;
    public String Location;
    public String Tagline;
    public String TwitterSearch;
    public Date StartDate;
    public Date EndDate;

    public ConferenceEditableInfo() {
    }

    public ConferenceEditableInfo(String name, String description, String location, String tagline, String twitterSearch, Date startDate, Date endDate) {
        Name = name;
        Description = description;
        Location = location;
        Tagline = tagline;
        TwitterSearch = twitterSearch;
        StartDate = startDate;
        EndDate = endDate;
    }
}
